### PR TITLE
Frequentist (RM) AN(C)OVA rewrites 2

### PR DESCRIPTION
**Pull request by JohnnyDoorn**
Now after a rebase as suggested by Tim

### CODE REVIEW

**Comment by JohnnyDoorn** [comment]
@TimKDJ what went wrong here? I rebased, reinstalled the JASP packages, reran the manageTestPlots and saved the new plots, then added those to this PR... but Travis still says they are different

**Comment by TimKDJ** [comment]
I'll have a look, can you tell me your versions of:
- jaspResults
- JASPgraphs
- jasptools
- vdiffr
- ggplot2

File: Resources/ANOVA/qml/AnovaRepeatedMeasures.qml [hunk 1]
```diff
@@ -22,7 +22,7 @@ import JASP.Widgets 1.0
 
 Form
 {
-	usesJaspResults: false
+	usesJaspResults: true
```
Comment (vandenman) [inline Resources/ANOVA/qml/AnovaRepeatedMeasures.qml]:
```suggestion
```
This is the default, so you can just omit this.

File: JASP-Tests/R/tests/figs/deps.txt [hunk 2]
```diff
@@ -1,3 +1,3 @@
 - vdiffr-svg-engine: 1.0
-- vdiffr: 0.3.1
+- vdiffr: 0.3.0
```
Comment (vandenman) [inline JASP-Tests/R/tests/figs/deps.txt]: You downgraded `vdiffr`?

File: JASP-Tests/R/tests/testthat/test-ancova.R [hunk 3]
```diff
@@ -376,22 +328,29 @@ test_that("Field - Chapter 6 results match", {
...
   expect_equal_plots(plot1, "PuppyLove1", "Ancova")
-  
-  plot2 <- results[['state']][['figures']][[2]][["obj"]]
+
+  plot2 <-  results$state$figures[[2]]$obj
   expect_equal_plots(plot2, "PuppyLove2", "Ancova")
-  
-  plot3 <- results[['state']][['figures']][[3]][["obj"]]
+
+  plot3 <-  results$state$figures[[3]]$obj
```
Comment (vandenman) [inline JASP-Tests/R/tests/testthat/test-ancova.R:354]: these changes are unnecessary?

File: JASP-Tests/R/tests/testthat/test-anova.R [hunk 4]
```diff
@@ -201,8 +207,8 @@ test_that("Q-Q plot matches", {
   options$fixedFactors <- "contBinom"
   options$modelTerms <- list(list(components="contBinom"))
   options$qqPlot <- TRUE
-  results <- jasptools::run("Anova", "test.csv", options)
-  testPlot <- results[["state"]][["figures"]][[1]][["obj"]]
+  results <- jasptools::run("Anova", "test.csv", options, view = FALSE)
```
Comment (vandenman) [inline JASP-Tests/R/tests/testthat/test-anova.R]:
```suggestion
  results <- jasptools::run("Anova", "test.csv", options)
```
view argument is redundant (here and in the other calls).

**Review by vandenman — CHANGES_REQUESTED** [review]
### ANOVA / ANCOVA
Looks good! Below a small list of things I found. I still need to take a look at the repeated measures and at the code itself though.

Bugs:

- clicking `Vovk-Sellke maximum p-ratio removes the main table
image

Minor comments:

- I can enter contBinom as covariate.

- the different number of bootstraps looks a bit odd.
image

**Pull request closed without merging**